test/functionalTest/cases/1259_new_json_for_v2/put_v2_entities_eid_attrs_attr.test

```diff
@@ -0,0 +1,206 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+PUT /v2/entities/eid/attrs/attr
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB 0-255
+
+--SHELL--
+
+#
+# 01. PUT /v2/entities/E1/attrs/A1 and see it fail
+# 02. POST /v2/entities, creating E1 with A1
+# 03. PUT /v2/entities/E1/attrs/A1 and see it work
+# 04. GET /v2/entities/E1/attrs/A1
+# 05. PUT /v2/entities/E1/attrs/A1 'old way' - metadatas not inside 'metadata: {}' - see it fail
+# 06. PUT /v2/entities/E1/attrs/A1 'new way' - metadatas inside 'metadata: {}' - see it work
+# 07. GET /v2/entities/E1/attrs/A1 to see that 06 worked
+#
+
+echo "01. PUT /v2/entities/E1/attrs/A1 and see it fail"
+echo "================================================"
+payload='{
+  "value": 12
+}'
+orionCurl --url /v2/entities/E1/attrs/A1 --payload "$payload" -X PUT --json
+echo
+echo
+
+
+echo "02. POST /v2/entities, creating E1 with A1"
+echo "=========================================="
+payload='{
+  "id": "E1",
+  "type": "T1",
+  "A1": "a1"
+}'
+orionCurl --url /v2/entities --payload "$payload" --json
```

Comment:
Exactly.
See description of issue #1259:
7. Selector to make the broker accept different input payloads for /v2/entities

This change came in after I started this work, so, I prefer to first finish what I started and after that make the new changes.

NTC (for this PR)